### ISSUE TITLE
Highlighting for footnotes is strange

### ISSUE THREAD

**Issue by hhau** [REditorSupport/sublime-ide-r#19]
The feedback from the linter is not quite right for syntactically valid footnotes, see the attached screenshot.

![screen shot 2018-10-23 at 08 41 13](https://user-images.githubusercontent.com/14539534/47344206-d2d33000-d69f-11e8-94fc-c7b970effac9.png)

The highlighting under the LaTeX is #4, but I didn't spot an issue about footnotes. Let me know if you need any more info from me. 

Simplest case is a blank .rmd file with:

```
Some text [^1].

[^1]: Some text that shouldn't be highlighted.
```
in it.

**Comment by randy3k** [comment]
Thanks, I could reproduce it. Unfortunately, I am quite busy these days to fix this issue. I would appreciate any help from the community.

**Comment by randy3k** [comment]
should be fixed in v0.1.7
